.changeset/corks-are-silly.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/catalog-client': minor
+---
+
+Create an initial CatalogClient implementation.
```

Comment:
Skip this changeset entry completely imo, since we're introducing the package here. Otherwise we'll have a version bump of the package that "introduces" it x). We could also keep it but make it a patch I guess ^_>

Reply:
oh for sure, deleting is probably best